main.js

```diff
@@ -4,22 +4,39 @@ var bodyInput = document.querySelector('.form-textarea-body');
 var submit = document.querySelector('.form-button-save');
 var cardSection = document.querySelector('.section-cards');
 var form = document.querySelector('.form');
-var allCards = [];
 var buttonOpen = document.querySelector('.nav-button-open');
 var navSection = document.querySelector('.nav-section');
 var image = document.getElementById('menu');
+var allCards = [];
 
 submit.addEventListener('click', submitNewIdea);
+titleInput.addEventListener('keyup', enableButton);
+bodyInput.addEventListener('keyup', enableButton);
+window.onload = loadCards;
+cardSection.addEventListener('click', function() {
+  cardEvent(event.target);
+});
+buttonOpen.addEventListener('click', menuDropdown);
 
 function submitNewIdea() {
+  var card = createCardObject();
+  createCard(card);
+  formReset();
+};
+
+function createCardObject() {
   var titleValue = titleInput.value;
   var bodyValue = bodyInput.value;
-  var userCard = new Idea(titleValue, bodyValue);
-  //Should change var userCard to var newIdea or something similar--should follow suite with idea.js and class Idea
+  var cardObj = {
+    id:Date.now(),
+    title:titleValue,
+    body:bodyValue,
+    starred:false,
+  };
+  var userCard = new Idea(cardObj);
```

Comment:
the back and forht between cardObj, userCard, and Idea is a little offputting here. Don't really have a suggestion, just a thought